src/ndarray/ndarray_function.cc

```diff
@@ -183,5 +184,22 @@ void ElementwiseSum<cpu>(mshadow::Stream<cpu>* s,
   }
 }
 
+/*
+ * \brief Enables use of function defined under Dequantize2Bit operator for an ndarray
```

Comment:
?

Reply:
I needed to use these function defined in the operator (src/operator/contrib/) in ndarray.cc 
I was unable to refer to that function and get it to compile/link successfully.
I did this similar to how CastStorageDispatch is defined in src/common/utils.cc